apps/api/src/main.ts

```diff
@@ -19,6 +20,14 @@ async function bootstrap(): Promise<void> {
   const app = await NestFactory.create(AppModule, {
     logger: loggerConfig,
   });
+  const config = new DocumentBuilder()
+    .setTitle('OsmoX')
+    .setDescription('The OsmoX API description')
```

Comment:
This should be an actual description not just the sentence. Get reference description reference from pacakge.json or readme.

Reply:
updated in https://github.com/OsmosysSoftware/osmo-x/pull/191/commits/287401c7d3b3b33e068c3cdbe6ce080689544635